README.md

```diff
@@ -127,14 +160,27 @@ To count planes easily
 Planes that have landed must be at an airport
 ```
 
-### Domain Models
+**Initial thoughts:**
+
+1. Make a new Weather class.
+2. Default weather in _Airport_ is always **Sunny**
+3. Can pass in a Weather with **Stormy** as parameter when instantiating a new _Airport_.
+4. Generate a random number between 1 and 2, assigning each to either **Sunny** or **Stormy**.
+   6
+
+_Test 13_ - Check if both **Sunny** and **Stormy** are passed in correctly to Airport.\
+_Test 14_ - Land a plane with weather set as **Sunny**.\
+_Test 15_ - Land a plane with random weather set using `getRandomWeather()` in _Airport_.\
+_Test 16_ - Take off a plane with random weather set using `getRandomWeather()` in _Airport_.
+
+### Final Domain Model
 
 | Objects | Properties                        | Messages               | Outputs  |
 | ------- | --------------------------------- | ---------------------- | -------- |
 | Airport | capacity @Number                  | setCapacity(@Number)   | @Void    |
 |         | airportWeather @Object {@Weather} |                        | @Weather |
 |         | planesInAirport @Array[@Plane]    | isAirportFull()        | @Boolean |
-|         |                                   | landPlane(@Plane)      | @String  |
+|         | airportWeather @String            | landPlane(@Plane)      | @String  |
 |         |                                   |                        | @Void    |
 |         |                                   | takeOffPlane(@Plane)   | @String  |
 |         |                                   |                        | @Void    |
```

Comment:
Should've also added another method called, getSumofAirportPlanes(), to return the sum of planes that exists in the airport to satisfy the last user story in the extended criteria. But this can also be achieved by using planesInAirport.length as that landPlane only allows unique planes to land.